operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -714,6 +733,29 @@ private boolean hasExceededRetryCount() {
           >= DomainPresence.getDomainPresenceFailureRetryMaxCount();
     }
 
+    private void disableOnlineUpdateIfSpecChgNotCompatible(DomainPresenceInfo cachedInfo) {
+      DomainSourceType domainSourceType = Optional.ofNullable(liveInfo)
+          .map(DomainPresenceInfo::getDomain)
+          .map(Domain::getDomainHomeSourceType)
+          .orElse(DomainSourceType.Image);
+
+      // For MII, reset the onlineUpdate.enabled to false if changes in the spec involves more than introspectVersion
+      // and onlineUpdate, disable online update ??
+
+      if (domainSourceType.equals(DomainSourceType.FromModel) && !isSpecChgOk4OnlineUpdate(liveInfo, cachedInfo)) {
+        LOGGER.info("DomainType is FromModel and Online changes requested in the spec involves more than "
```

Comment:
This info message looks a little cryptic to me - lets meet to word smith it.  

For example, we could make it clear the requested changes are still going to occur, and we could mention that running pods will be rolled in order to propagate the changes.   Something like "Overriding any potential Model In Image online updates to offline updates plus a roll because additional attributes beyond the introspectVersion attribute changed in the domain resource.".  

Also, it looks like this log message can be often spurious?  I don't understand how the Operator can know there are any on-line updates until it runs the Operator...

 (Maybe this should be a 'fine' instead of an 'info' regardless?)

Reply:
Propose changing this to:
```
+        LOGGER.info("The domain 'spec.domainHomeSourceType' is 'FromModel' and "
+            + "'spec.configuration.model.onlineUpdate.enabled' is 'true', "
+            + "but there are changes in the domain "
+            + "resource that require a domain roll "
+            + "(for example, image name, serverPod environment, ...). "
+            + "Any WebLogic configuration changes will use an offline update, "
+            + "and the domain will roll if the update is successful.");
```

Reply:
Closing - the overall flow changes so the log message was removed.